DemoApp/Screens/UserProfileViewController/UserProfileViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+class UserProfileViewController: UIViewController, CurrentChatUserControllerDelegate {
+    private let imageView = UIImageView()
+    private let nameTextField = UITextField()
+    private let updateButton = UIButton()
+
+    let currentUserController: CurrentChatUserController
+
+    init(currentUserController: CurrentChatUserController) {
+        self.currentUserController = currentUserController
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .white
+
+        [imageView, nameTextField, updateButton].forEach {
+            view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.cornerRadius = 30
+        imageView.layer.masksToBounds = true
+        updateButton.setTitle("Update", for: .normal)
+        updateButton.backgroundColor = .blue
+        updateButton.addTarget(self, action: #selector(didTapUpdateButton), for: .touchUpInside)
+        updateButton.isHidden = !DemoAppConfiguration.isStreamInternalConfiguration
```

Comment:
Nit: Would be nice for the button that has the same design as the one below so that we have the same consistent across the Demo App. Since customers usually use this app, it is nice that it looks good all around 😄 

<img width="168" alt="image" src="https://github.com/GetStream/stream-chat-swift/assets/12814114/5413f594-8726-4eb5-8b95-2a9f01c02c30">

I believe that is as easy as changing the initialization to this:
```swift
 UIButton(configuration: .filled(), primaryAction: nil)
```